### DISCUSSION TITLE
Question: Possible to point to directory containing xml files?

### DISCUSSION THREAD

**Discussion by myleshyson** [engram-design/FeedMe#916] [Ideas]
I was wondering if it's possible to point the feed path to a local directory that's got a bunch of xml files in it, or if that will become available at some point in the future? We have a subscription to xml content but we can't control how we receive it, so we end up getting like a folder of 3000 xml files. We're thinking about coming over to Craft and currently in another system we are able to loop through those files and import them.

**Comment by kristiansp** [comment]
I have no idea if something like this would be available in the future, but if they have the same structure, you could perhaps make use of the [pagination URL](https://docs.craftcms.com/feed-me/v4/feature-tour/primary-element.html#pagination-url) to point to the next feed. Possibly you could hook onto one of the events and do this dynamically.

**Comment by kristiansp** [comment]
Another option would be to set the URL parameter when triggering your feeds, so that you make a service to loop through your files and feed them on to Feed Me:
[https://docs.craftcms.com/feed-me/v4/feature-tour/trigger-import-via-cron.html#parameters](https://docs.craftcms.com/feed-me/v4/feature-tour/trigger-import-via-cron.html#parameters)

**Comment by myleshyson** [comment]
Hey so I ended up taking your advice and using `sed` to create a `nextPage` tag in each file so that I could use feedme's pagination. I still think being able to just point it to a directory would be nice but using `sed` or looping through with the cron is fine for us now. Thanks!

**Comment by kristiansp** [comment]
@myleshyson Happy to hear it worked out! I see your point, and it doesn’t sound super hard to implement. I guess part of the issue is that a folder potentially could contain anything, both XMLs with different data/structure, or basically any other files or subfolders. It would kind of be good practice that data that belongs to each other have _some_ sort of relation to each other, other than being in the same folder. But sometimes you are in the situation you are whew you get what you get, and have to deal with it.
